config/install/acquia_lift.settings.yml

```diff
@@ -17,4 +17,5 @@ thumbnail: []
 visibility:
   # Track, only when the page URL falls OUTSIDE OF these path patterns.
   path_patterns: "/admin\n/admin/*\n/batch\n/node/add*\n/node/*/*\n/user/*/*\n/block/*"
+content_replacement_mode: 'untrusted'
```

Comment:
It's not "deployment" of your content.  Particularly when we get into the realm of content hub, etc. this could lead to a lot of confusion.  It is how your content will be "replaced" during personalization.  I can see the case for nesting and I'll go ahead and add. In terms of the word "default", we could get into all sorts of semantics of "default" vs. "site-wide" etc.  That kind of thing is for the user-facing text.  For the underlying value in terms of the variable it seems that "default_" is extra noise.  There is the mode and then it can be overridden, but it's not that there is a default in the system and then different situations that lead to different modes.  I guess I could be convinced if you really feel strongly, but in terms of labelling I'll address those below with your other comment.